core/src/main/scala/bekindrewind/RecordOptions.scala

```diff
@@ -1,7 +1,12 @@
 package bekindrewind
 
-final case class RecordOptions(notRecordedThrowsErrors: Boolean, overwriteAll: Boolean) {
+import scala.concurrent.duration.FiniteDuration
 
+final case class RecordOptions(
+  notRecordedThrowsErrors: Boolean,
+  overwriteAll: Boolean,
+  expiresAfter: Option[FiniteDuration]
```

Comment:
This works, but I'd prefer to use `java.time` exclusively (`java.time.Duration` in this case).

Reply:
Changed to `java.time.Duration` in https://github.com/reibitto/be-kind-rewind/pull/18/commits/6d3d8ee5fb2dea9571c1772dd47daeb31e3fa4a6